scripts/asr_language_modeling/ngram_lm/ngram_merge.py

```diff
@@ -0,0 +1,454 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
here and in other new files: NVIDIA CORPORATION -> NVIDIA CORPORATION & AFFILIATES

Reply:
done